keps/sig-network/4444-service-traffic-distribution/README.md

```diff
@@ -206,11 +209,12 @@ while making routing decisions. It does not offer strict routing guarantees.
 
 The field will support the following initial values:
 
+* `PreferClose`: Indicates a preference for routing traffic to endpoints in
+  the same zone as the client.
```

Comment:
Optionally, clarify what zones are.
```suggestion
  the same zone as the client. A zone represents a logical failure domain and has the same meaning as
  in the well-known label `topology.kubernetes.io/zone`.
```

Reply:
Thanks Tim! I added this under the kube-proxy implementation section:

> #### `PreferClose`
> * **Meaning:** Attempts to route traffic to endpoints within the same zone as
>   the client. A zone represents a logical failure domain and has the same
>   meaning as in the well-known label `topology.kubernetes.io/zone`. If no
>   endpoints are available within the zone, traffic would be routed to other
>   zones.